hack/get-project-maintainers.sh

```diff
@@ -27,4 +27,4 @@ cd "${REPO_ROOT}" && make ${YQ_BIN} >/dev/null
 
 KEYS=()
 while IFS='' read -r line; do KEYS+=("$line"); done < <(${YQ_PATH} e '.aliases["cluster-api-maintainers"][]' OWNERS_ALIASES)
-echo "${KEYS[@]/#/@}"
+echo "${KEYS[@]/#/@} @cluster-api-release-team"
```

Comment:
I think this won't work cross-org (kubernetes => kubernetes-sigs) assuming the GItHub team is created in kubernetes-sigs (?)

Can we test this manually on one of our old PRs once the GitHub team is created and then modify here accordingly?

Reply:
If we don't find a way to easily do this, I think it's fine to mention folks manually or eventually put the release team info in a format where we can also grep for GitHub IDs (might get a bit hacky)

Reply:
Hm thinking about it some more. Could be beneficial to have the release team info in some structured format (YAML with arrays?) anyway to:
* easily open the PR for the Slack group
* easily open the PR for the GitHub team

I wouldn't go so far as to automatically generate those 2 PRs, but for the bump PR it should be realistic.

Reply:
we might want an OWNERs file for release notes as per #7128 so in that case having the release team defined in OWNERS_ALIASES would be useful

Reply:
Hm, not sure if this needs further discussion. At this point I'm not sure if we should give the entire release team including shadows access to create branches / tags / releases via OWNERS.

Reply:
Additional note. If we put the list of release team members into the repo we always have to keep it up-to-date on main and all maintained branches as usually the promotion PR is generated when the current "release" branch (which could also be main) is checked out.

I don't know how this would change with #7128

Reply:
Would we add the entire release team to the release notes OWNERS file?

Reply:
not necessarily, we could be more granular if we wanted to and have a different OWNERs group for each release team role

Reply:
Sounds fine to me in general, but I guess we have to figure out the details of who exactly gets which reviewer or approver (?) rights.

Reply:
I suggest having a group for the release team leads and one for the release team shadows; both gets the email, the first approve